Views/Shared/RecursiveNavigation.cshtml

```diff
@@ -0,0 +1,23 @@
+@model Piranha.Models.SitemapItem
+
+@if (@Model.Items.Count > 0)
+{
+    <li class="@(Model.ParentId == null ? "nav-item dropdown" : "")">
```

Comment:
We don't want class="". Please fill in the whole class=nav-item or nothing.

Reply:
Done